package.json

```diff
@@ -30,6 +30,7 @@
     "storybook": "start-storybook -p 6006"
   },
   "dependencies": {
+    "lodash": "^4.17.5",
```

Comment:
maybe `lodash.omit`?

Reply:
this is something I wanted to discuss in the meeting :)